docs/contributor-docs/contributing.md

```diff
@@ -24,7 +24,7 @@ Documentation is incredibly helpful! All these docs are written using `mkdocs` v
 
 ### Translations
 
-We're still working to set up a translation system that will help automate this, but for now translations occur in the frontend code. This is a complex process so please ask for help if you wish to assist right now.
+If you're interested in helping to translate Jellyfin into your local language, we use [Weblate](https://weblate.org/en/) running at [translate.jellyfin.org](https://translate.jellyfin.org) to handle translations. These are collected in the `translations` branch of the [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repository and are merged into master before each release.
```

Comment:
```suggestion
If you're interested in helping to translate Jellyfin into your local language, we use [Weblate](https://weblate.org/en/) running at [translate.jellyfin.org](https://translate.jellyfin.org) to handle translations. These are collected in the [translations](https://github.com/jellyfin/jellyfin-web/tree/translations) branch of the [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repository and are merged into master before each release.
```
might also want to indicate that this is for Web UI only _for now_, and more are to come.

Reply:
I tweaked this up a bit more now that it's not just `jellyfin-web` being translated and made it repo-generic..